src/python/finalize/cli/main.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
-	"io"
-	"io/ioutil"
-	"os"
 	"github.com/cloudfoundry/python-buildpack/src/python/finalize"
 	_ "github.com/cloudfoundry/python-buildpack/src/python/hooks"
 	"github.com/cloudfoundry/python-buildpack/src/python/pyfinder"
+	"github.com/cloudfoundry/python-buildpack/src/python/requirements"
```

Comment:
minor: looks like these could use some re-sorting

Reply:
I used the default sort of GoLand, any suggestions on how to modify it?

Reply:
Hmm... I use intelliJ, and as long as you have groups of imports with blank lines in between groups, it seems to sort appropriately.

Format:

```
<go standard libs>
...

<third-party libs>
...

<first-party libs>
...
```

Example:

```go
import (
	"io"
	"io/ioutil"
	"os"
	"time"

	"github.com/cloudfoundry/libbuildpack"

	"github.com/cloudfoundry/python-buildpack/src/python/finalize"
	_ "github.com/cloudfoundry/python-buildpack/src/python/hooks"
	"github.com/cloudfoundry/python-buildpack/src/python/pyfinder"
	"github.com/cloudfoundry/python-buildpack/src/python/requirements"
)
```

this grouping format seems to be the recommended practice for go